doc/api/path.md

```diff
@@ -200,7 +209,7 @@ path.extname('.index.md');
 // Returns: '.md'
 ```
 
-A [`TypeError`][] is thrown if `path` is not a string.
+A [`TypeError`][] is thrown if `path` is not a string or an instance of \[`URL`]\[].
```

Comment:
```suggestion
A [`TypeError`][] is thrown if `path` is not a string or an instance of [`URL`][].
```
Backslashes are not needed, the linter just wanted reference at the end of document